### PR TITLE
Multiselect Cleanup

### PR DESCRIPTION
**Pull request by cwmanning**
Fixes multiselect backwards-compatibility. Pre-existing 'single' values are converted to an array for more consistent handling.

### CODE REVIEW

File: src/components/__tests__/ars-test.jsx [hunk 1]
```diff
@@ -13,11 +13,16 @@ describe("Ars", function() {
...
-    let component = Test.renderIntoDocument(makeComponent({ onChange }))
+    let picked = 9
+    let component = Test.renderIntoDocument(makeComponent({ onChange, picked }))
 
     it ("has a selection component", function() {
       component.refs.should.have.property("selection")
     })
+
+    it ("migrates a single `picked` value to an array", function() {
+      component.state.should.have.property("picked").that.deep.equals([9])
```
Comment (nhunzaker) [inline src/components/__tests__/ars-test.jsx]: what about `.that.deep.equals([picked])`?

Comment (nhunzaker) [inline src/components/__tests__/ars-test.jsx]: Also... woah I had no idea about `that`, or `deep`. I've always done:

`component.state.should.have.property("picked").should.eql([9])`

Yours is much cleaner!

File: src/components/ars.jsx [hunk 2]
```diff
@@ -25,9 +25,14 @@ let Ars = module.exports = React.createClass({
   },
 
   getInitialState() {
+    let { picked } = this.props
+    if (picked && typeof picked !== 'object') {
+      picked = [picked]
+    }
```
Comment (nhunzaker) [inline src/components/ars.jsx]: Is this checking for an array? if so, what about `Array.isArray(picked)` ?

**Comment by nhunzaker** [comment]
:+1:

File: src/components/__tests__/ars-test.jsx (hunk 1 above)
Comment (cwmanning) [inline src/components/__tests__/ars-test.jsx]: Oh right, I like using `that` here. I found that `deep` was necessary on arrays in some earlier testing. When I used `eql` I got messages like `Expected [ 0 ] to be equal to [ 0 ]`.

File: src/components/ars.jsx (hunk 2 above)
Comment (cwmanning) [inline src/components/ars.jsx]: Yes. That's much better.

**Pull request merged**